bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/common/CommerceIdentifier.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright 2021 Adobe. All rights reserved.
+ *
+ *   This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package com.adobe.cq.commerce.core.components.models.common;
+
+/**
+ * An identifier for any commerce entity (category or product). A commerce entity can be identified by more than one key, for example a
+ * product can be identified by the URL key or by the SKU.
+ */
+public interface CommerceIdentifier {
+
+    /**
+     * The type of this identifier
+     */
+    enum IdentifierType {
+        SLUG, URL_KEY, ID
```

Comment:
`SLUG` is basically `URL_KEY` for products, was this meant to be `SKU, URL_KEY, ID`? We'd have to decide if we use `SLUG` or `URL_KEY` : I'd go for `URL_KEY` because this is already what we use in `UrlProvider`.

Reply:
It was `SKU`, sorry. I'll update it